test/interface/scoregradelbo.jl

```diff
@@ -9,7 +18,24 @@ using Test
 
     (; model, μ_true, L_true, n_dims, is_meanfield) = modelstats
 
-    q0 = TuringDiagMvNormal(zeros(Float64, n_dims), ones(Float64, n_dims))
+    q0 = MeanFieldGaussian(zeros(n_dims), Diagonal(ones(n_dims)))
+
+    @testset "basic" begin
+        @testset for adtype in AD_scoregradelbo_interface, n_montecarlo in [1, 10]
+            obj = ScoreGradELBO(n_montecarlo)
+            _, _, stats, _ = optimize(
+                rng,
+                model,
+                obj,
+                q0,
+                10;
+                optimizer=Descent(1e-5),
+                show_progress=false,
+                adtype=adtype,
+            )
+            @assert isfinite(last(stats).elbo)
```

Comment:
Same question here, do we check the correctness of the result somewhere?

Reply:
same as bove